partcad-cli/src/partcad_cli/click/commands/convert.py

```
@@ -0,0 +1,15 @@
+import rich_click as click
+import partcad as pc
+from partcad.actions.part_actions import convert_part_action
+
+@click.command(help="Convert parts to another format and update their type.")
+@click.argument("part_name", type=str, required=True)
+@click.option("-t", "--target-format", help="Target format for conversion.", type=click.Choice(["step", "brep", "stl", "3mf", "threejs", "obj", "gltf"]), required=True)
+@click.pass_obj
+def cli(ctx, part_name, target_format):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)